src/assets/images/cross.svg

```diff
@@ -1,5 +1,5 @@
 <!-- Generated by IcoMoon.io -->
-<svg version="1.1" xmlns="http://www.w3.org/2000/svg" fill='white' width="32" height="32" viewBox="0 0 32 32">
+<svg version="1.1" xmlns="http://www.w3.org/2000/svg" width="32" height="32" viewBox="0 0 32 32">
```

Comment:
you can use `fill=currentColor` and svg will have the same color as parent font. (it inherits from `color` property)